tests/testthat/_snaps/space_filling.md

```diff
@@ -22,6 +22,17 @@
       x This argument contains unknowns: `mtry`.
       i See the `dials::finalize()` function.
 
+---
+
+    Code
+      grid_max_entropy(mixture(), levels = 11)
+    Condition
+      Warning:
```

Comment:
This warning should either be an error right away, or if we really want people to be able to continue with this mix-up, we should remove the stray argument so it doesn't interfer like it does here, causing an error later on with a surprising error message. This applies to all other grid functions with this type of warning as well. I'll turn it into a separate issue.